ion/src/simulator/init.cpp

```diff
@@ -116,3 +116,19 @@ void Ion::msleep(long ms) {
     }
   }
 }
+
```

Comment:
Couldn't we simply reuse the STL-based code from blackbox?

Reply:
In the simulator there are calls to 
  sDisplay->redraw();
  Fl::wait(0);
Without them, the content of the window is not updated in the while sleeping

Reply:
Why would the window need to be updated *while* sleeping? There are no threads in Epsilon, so there's no reason for the window to change, is there?

Reply:
It's to avoid this : 

![image](https://user-images.githubusercontent.com/1298609/48550383-dccd0500-e8d2-11e8-8577-9c3494360cb9.png)

This problem may be hidden by a compositor (on linux at least), which will keep the last state of the window.

This also allows to interrupt sleeping with the back key.